### ISSUE TITLE
color contrast issue of placehold text

### ISSUE THREAD

**Issue by zhenwan** [carbon-design-system/carbon#2940]
the placeholder color defined for bx--search-input class causes issue  (see attachment) 

.bx--search-input::placeholder {
    color: #8c8c8c;
    font-weight: 400; }

<img width="482" alt="Placeholder color for Input" src="https://user-images.githubusercontent.com/14806866/58991094-0f761680-87ad-11e9-9742-9cce3356e962.png">

**Comment by dakahn** [comment]
Thanks for reporting. Can you fill out the template provided for a11y bugs?

**Comment by zhenwan** [comment]
## Title line template: [Title]: Brief description
the placeholder color defined for bx--search-input class causes issue

## Environment
All

> Operating system
All

> Browser
All

> Assistive technology used to verify
Colour Contrast Analyser

## Detailed description

> What version of the Carbon Design System are you using?
7.1.0

> What did you expect to happen?

> What happened instead?
The current contrast ratio is: 3.4:1

"The visual presentation of text and images of text has a contrast ratio of at least [4.5:1"](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)


> What WCAG 2.1 checkpoint does the issue violate?

[Placeholder Research](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Placeholder_Research



> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
- Notes

**Comment by dakahn** [comment]
Thank you for taking the time to report 👍 

Placeholder text should always be redundant and unnecessary. **It should always be accompanied by a label.** That label should be visible and meet WCAG AA contrast requirements.

You'll notice here that any example input with placeholder text has a corresponding label. 
http://react.carbondesignsystem.com/?selectedKind=Form&selectedStory=Default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel

reference https://github.com/carbon-design-system/design-system-website/issues/166
